roles/openshift_manage_node/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openshift_manage_node_master_host: "{{ groups.oo_first_master.0 }}"
```

Comment:
Same comment here as with the etcd roles, we shouldn't be relying on specific group names in the roles.